### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by Roshanjossey**
This document defines how we engage with the community. It states that this is a welcoming and inclusive project.
This is added to bring this project to GitHub's recommended community standards

### CODE REVIEW

**Comment by Roshanjossey** [comment]
I've added my e-mail Id as contact for reporting any issues there. I'd like to change it to a better suited mail

**Comment by tallpsmith** [comment]
Where does your email get used in this context exactly?  I can't see it in the Code of Conduct file.

Can you help me by pasting the link to Github here where this is recommended?  I am certainly not against the philosophy of the code of conduct of course, just want to see where this has come from.

tah.

Paul

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at rjossey@aconex.com. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (Roshanjossey) [inline CODE_OF_CONDUCT.md]: @tallpsmith, email is specified in this line. It's for reporting any incident that violates this code of conduct

**Comment by Roshanjossey** [comment]
@tallpsmith, I've pointed out the line which has email-id above. This is a link to GitHub's community standards checklist for scrutineer https://github.com/Aconex/scrutineer/community

More about code of conduct https://help.github.com/articles/adding-a-code-of-conduct-to-your-project/

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (tallpsmith) [reply CODE_OF_CONDUCT.md]: i am so blind...

You could perhaps ask CIT to create an email alias `scrutineer@aconex.com` that maps to yours and my email address for now?

Comment (Roshanjossey) [reply CODE_OF_CONDUCT.md]: Will do. Thanks

Comment (Roshanjossey) [reply CODE_OF_CONDUCT.md]: I'm thinking about a mail id for all open source projects like opensource@aconex.com. How about that?

**Pull request merged**